static/downloadables/parse_associations.py

```diff
@@ -238,7 +248,7 @@ def parse_txt(filteredFilePath, clumpsObjDict, tableObjDict, snpSet, clumpNumDic
                             # The snp wasn't in the clump map (meaning it wasn't in 1000 Genomes), so add it
                             else:
                                 sample_map[snp] = alleles
-                        else:
+                        elif riskAllele not in alleles and isIndividualClump:
```

Comment:
Does this cover all the cases? I'm just worried if we will be missing something by not having an 'else'

Reply:
I'm pretty sure it covers all cases, but I'll change it back to 'else' just for cleanliness. I was writing it out more just so I could see the logic myself.